cypress/support/step-definitions/text-area-steps.js

```diff
@@ -9,6 +9,7 @@ import {
 
 const TEXT_ALIGN = 'text-align';
 const TEXTAREA_CLASS = 'carbon-textarea__input';
+export const DEBUG_FLAG = false;
```

Comment:
`DEBUG_FLAG` should be imported instead of declaring new variable
`import { DEBUG_FLAG } from '..';`

Reply:
fixed